### PR TITLE
Switch to tox

### PR DESCRIPTION
**Pull request by stephendonner**
Work in progress; should be close, though.  Would love feedback!

### CODE REVIEW

**Comment by stephendonner** [comment]
Additional notes:

1. We probably want and need to update this and every repo's README documentation, not only due to recent the introduction of tox, but also due to Web QA -> Firefox Test Engineering
2. As mentioned in https://github.com/mozilla/mozillians-tests/issues/248, 15 or so tests are being skipped on dev + staging.
3. I ran with tox -e tests -- -n=2  --base-url https://mozillians.allizom.org --variables=/Users/sdonner/Desktop/mozillians.json

**Comment by stephendonner** [comment]
Note to self:
1. Remove virtualenv and virtualenvwrapper items
2. Fix duplicate Running Tests headings

File: README.md [hunk 1]
```diff
@@ -117,45 +117,12 @@ with varying privilege levels.
...
-"~" will not resolve to the home directory when used in the py.test command line.
-
-Some options for py.test are pre-specified by the file mozillians-tests/mozwebqa.cfg
-
-The mozwebqa plugin has advanced command line options for reporting and using
-browsers. See the documentation on [pytest mozwebqa github][pymozwebqa].
-[pymozwebqa]: https://github.com/mozilla/pytest-mozwebqa
-
-__Troubleshooting__
+Running Tests
```
Comment (davehunt) [inline README.md]: Remove the additional heading, and there's more we can remove above about installing dependencies and virtualenvs.

File: setup.cfg [hunk 2]
```diff
@@ -2,5 +2,9 @@
 ignore=E501
 
 [tool:pytest]
+python_files=test_*.py
```
Comment (davehunt) [inline setup.cfg]: We can remove this line.

File: README.md (hunk 1 above)
Comment (stephendonner) [inline README.md]: Done (both)

File: setup.cfg (hunk 2 above)
Comment (stephendonner) [inline setup.cfg]: Done, self!  ;-P

Comment (stephendonner) [inline setup.cfg]: Done!

**Comment by stephendonner** [comment]
Final r?  @davehunt

**Pull request merged**